src/commands.ts

```diff
@@ -72,7 +73,7 @@ async function open ( file = false, page? ) {
 
     const editorPath = activeTextEditor.document.uri.fsPath;
 
-    filePath = editorPath ? editorPath.substring ( repopath.length + 1 ) : undefined;
+    filePath = editorPath ? editorPath.substring ( repopath.length + 1 ).replace( /\\/g, '/' ) : undefined;
```

Comment:
Part of me was hoping for a more robust fix. Maybe something utilizing https://nodejs.org/api/url.html#url_url_pathtofileurl_path

Reply:
If you find a cleaner solution feel free to use it, I've personally usually resorted to the regex solution too.